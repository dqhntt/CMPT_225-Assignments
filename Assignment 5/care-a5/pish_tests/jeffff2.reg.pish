a = 10;
b = a + 5;
c = b + a;

if b > a {a = a + 5;}

if b == a {b = b - 1;}

if (b + 1) * 4 == 60 and b * 2 > c {c = c * 10;}

if c / 10 == 25 or (a + b) * 10 > c {b = b + 1;}

if c / 10 == 25 and (a + b) / 10 == 3 {a = 1;}

print a;
print b;
print c;