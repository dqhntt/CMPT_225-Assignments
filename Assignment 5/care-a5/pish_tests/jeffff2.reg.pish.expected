1
15
250
